alfresco-googledrive-repo-enterprise/pom.xml

```diff
@@ -107,9 +107,6 @@
                                     <assembly>
                                         <descriptor>target-assembly.xml</descriptor>
                                     </assembly>
-                                    <buildOptions>
```

Comment:
For a local-only variant image, this may be reasonable for now ? 

Are there any other viable alternatives (eg. see JIRA comment regarding changing "experimental" option from false to true) ?